src/mocks.c

```diff
@@ -2,7 +2,7 @@
 #include "mocks.h"
 
 // Signature of library initialization function
-void init();
+void stderred_init();
```

Comment:
There's `reset_stderred()` in this file already, so `init_stderred()` would fit the _established_ pattern, just take a look at `init_mocks()`. Consider renaming.